x/concentrated-liquidity/lp.go

```diff
@@ -216,7 +218,7 @@ func (k Keeper) WithdrawPosition(ctx sdk.Context, owner sdk.AccAddress, position
 		if !anyPositionsRemainingInPool {
 			// Reset the current tick and current square root price to initial values of zero since there is no
 			// liquidity left.
-			if err := k.uninitializePool(ctx, pool.GetId()); err != nil {
+			if err := k.reinitializePool(ctx, pool.GetId()); err != nil {
```

Comment:
The old name was correct. It uninitializes the pool when all liquidity is withdrawn

Reply:
@p0mvn  Are we not setting back the pool to the initial state where the square root and the current tick is zero though?

Reply:
changed back to uninitialize!